client/lib/plans/constants.js

```diff
@@ -1,28 +1,58 @@
+import i18n from 'lib/mixins/i18n';
+
 export const plansList = {
-	'free_plan': {},
-	'value_bundle': {},
-	'business-bundle': {},
+	'free_plan': {
+		getTitle: () => i18n.translate( 'Free' ),
+		getPriceTitle: () => i18n.translate( 'Free for life' )
+	},
+	'value_bundle': {
+		getTitle: () => i18n.translate( 'Premium' ),
+		getPriceTitle: () => i18n.translate( '$99 per year' )
+	},
+	'business-bundle': {
+		getTitle: () => i18n.translate( 'Business' ),
+		getPriceTitle: () => i18n.translate( '$299 per year' )
+	},
 	'jetpack_free': {},
 	'jetpack_business': {}
 };
 
+const allPaidPlans = [
+	'value_bundle',
+	'business-bundle'
+];
+
 export const featuresList = {
 	'google-analytics': {
+		getTitle: () => i18n.translate( 'Google Analytics' ),
 		plans: [ 'business-bundle' ]
```

Comment:
Yes we have and this is precisely it :)
We have endpoint for that and bunch of helpers around endpoint data.

But we discussed it a bit with @mtias and we came to the conclusion that we need hardcoded data in Calypso because:
- Hardcoding offer will get rid of loading cards in `/plans`
- We will be able to fine-tune messaging and offering without dealing with API
- We will be able to A/B test

Once we fine-tune offering and messaging, we can move this data to API